src/frontend/src/pages/FinancePage/CreateReimbursementRequest.tsx

```diff
@@ -9,21 +9,30 @@ import { routes } from '../../utils/routes';
 import ReimbursementRequestForm, {
   ReimbursementRequestDataSubmission
 } from './ReimbursementRequestForm/ReimbursementRequestForm';
+import { useToast } from '../../hooks/toasts.hooks';
 
 const CreateReimbursementRequestPage: React.FC = () => {
   const { isLoading: createReimbursementRequestIsLoading, mutateAsync: createReimbursementRequest } =
     useCreateReimbursementRequest();
   const { isLoading: receiptsIsLoading, mutateAsync: uploadReceipts } = useUploadManyReceipts();
+  const toast = useToast();
 
   if (createReimbursementRequestIsLoading || receiptsIsLoading) return <LoadingIndicator />;
 
   const onSubmit = async (data: ReimbursementRequestDataSubmission): Promise<string> => {
-    const { reimbursementRequestId } = await createReimbursementRequest(data);
-    await uploadReceipts({
-      id: reimbursementRequestId,
-      files: data.receiptFiles.map((file) => file.file!)
-    });
-    return reimbursementRequestId;
+    try {
```

Comment:
same here, this is actually already try, catched where onSubmit is called so we don't need to do it here as well